pde/target.target

```diff
@@ -1,33 +1,35 @@
-        </location>
-        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target name="JDTLS.EXT" sequenceNumber="39">
+    <locations>
+        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+            <unit id="com.google.gson" version="2.7.0.v20170129-0911"/>
+            <unit id="com.google.gson.source" version="2.7.0.v20170129-0911"/>
+            <unit id="org.apache.commons.io" version="2.2.0.v201405211200"/>
+            <unit id="org.apache.commons.lang3" version="3.1.0.v201403281430"/>
+            <repository location="https://download.eclipse.org/tools/orbit/R-builds/R20200529191137/repository"/>
+        </location>
+        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+            <unit id="org.eclipse.buildship.feature.group" version="0.0.0"/>
+            <repository location="https://download.eclipse.org/buildship/updates/e423/releases/3.x/3.1.6.v20220511-1359/"/>
+        </location>
+        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
             <unit id="org.eclipse.m2e.feature.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.m2e.logback.feature.feature.group" version="0.0.0"/>
-	        <repository location="https://download.eclipse.org/technology/m2e/snapshots/latest/"/>
-        </location>
-        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-            <unit id="org.eclipse.xtend.sdk.feature.group" version="0.0.0"/>
-            <unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
-            <repository location="https://download.eclipse.org/releases/2022-06/"/>
-        </location>
-        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-           <repository location="https://archive.eclipse.org/lsp4j/updates/releases/0.12.0/"/>
-           <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.0.0"/>
-        </location>
-        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-            <unit id="org.eclipse.jdt.ls.core" version="0.0.0"/>
-            <repository location="http://download.eclipse.org/jdtls/snapshots/repository/latest/"/>
-        </location>
-    </locations>
+	        <repository location="https://download.eclipse.org/technology/m2e/releases/1.20.1/"/>
```

Comment:
jdt.ls is using the latest m2e@2.x, is there any reason to change it back to 1.20.1 here?

Reply:
It’s to make it compatible with the eclipse update site we have here. But i can look into syncing with same information as in jdt.ls. Will do those changes in the other PR if you think its better since these changes are for resolving build issues.

Reply:
pde extension wouldn't package the m2e plugins into the final vsix. The m2e site is just used to resolve dependencies during compiling, it's ok to use the latest m2e.

I'm OK to fix it in this PR.

Reply:
Try to update to latest and I get the following

```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:2.7.3:validate-classpath (default-validate-classpath) on project org.eclipse.jdt.ls.importer.pde: Execution default-validate-classpath of goal org.eclipse.tycho:tycho-compiler-plugin:2.7.3:validate-classpath failed: org.osgi.framework.BundleException: Bundle org.eclipse.jdt.ls.importer.pde cannot be resolved:org.eclipse.jdt.ls.importer.pde [98]
[ERROR]   Unresolved requirement: Require-Bundle: org.eclipse.jdt.ls.core
[ERROR]     -> Bundle-SymbolicName: org.eclipse.jdt.ls.core; bundle-version="1.20.0.202302020244"; singleton:="true"
[ERROR]        org.eclipse.jdt.ls.core [80]
[ERROR]          Unresolved requirement: Require-Bundle: org.eclipse.m2e.core; resolution:="optional"
[ERROR]          Unresolved requirement: Require-Bundle: org.eclipse.m2e.jdt; resolution:="optional"
[ERROR]          Unresolved requirement: Require-Bundle: org.eclipse.m2e.maven.runtime; resolution:="optional"
[ERROR]          Unresolved requirement: Require-Bundle: org.eclipse.buildship.core; bundle-version="1.0.18"; resolution:="optional"
[ERROR]          Unresolved requirement: Require-Bundle: org.eclipse.lsp4j; bundle-version="[0.12.0,0.13.0)"
```

Reply:
it works for me with both m2e@latest and 1.20.1